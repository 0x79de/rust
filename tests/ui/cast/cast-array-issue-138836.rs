fn main() {
    let a: [u8; 3] = [1,2,3];
    let b = &a;
    let c = b as *const [u32; 3]; //~ ERROR mismatched types [E0308]
}
