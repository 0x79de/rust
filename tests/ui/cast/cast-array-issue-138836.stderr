error[E0308]: mismatched types
  --> $DIR/cast-array-issue-138836.rs:4:13
   |
LL |     let c = b as *const [u32; 3];
   |             ^^^^^^^^^^^^^^^^^^^^ expected `[u8; 3]`, found `[u32; 3]`
   |
   = note: expected array `[u8; 3]`
              found array `[u32; 3]`

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.
